config/jobs/kubernetes-sigs/cluster-api-provider-openstack/cluster-api-provider-openstack-presubmits.yaml

```diff
@@ -29,8 +29,10 @@ presubmits:
       preset-service-account: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/cluster-api:v20180720-d52d72975
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181010-2672d9494-master
```

Comment:
cluster-api image has etcd installed. this job basically does a unit test, switch to kubekins instead.